libs/utils/utils.js

```diff
@@ -116,11 +116,7 @@ export const [setConfig, getConfig] = (() => {
       } catch (e) {
         console.log("Invalid or missing locale:",e)
       }
-      if (config.contentRoot) {
```

Comment:
As I've seen it config.contentRoot is not being set anywhere, which makes this code redundant.

Reply:
Woah buddy... https://github.com/adobecom/stock/blob/main/pages/scripts/scripts.js#L28

Reply:
Ah, I see I was not thinking broadly enough 😅
Thanks Chris! I'll fix that